gulpfile.cjs

```diff
@@ -274,19 +273,6 @@ gulp.task("build-specs", function buildSpecs() {
           format: "iife",
         });
       })
-      .then(function () {
```

Comment:
I kept the SpecList.js file because I believe it's necessary for Specs.js to get created properly. Is it worth looking into avoiding this step since the file isn't used anywhere else (used to be used by the standalone SpecRunner too).